diff --git a/libtudor/download_driver.sh b/libtudor/download_driver.sh
#index 4450786..cb1dba9 100644
--- a/libtudor/download_driver.sh
+++ b/libtudor/download_driver.sh
@@ -9,8 +9,10 @@ mkdir -p "$TMP_DIR"
 
 #Download the driver executable and check hash
 INSTALLER="$TMP_DIR/installer.exe"
-wget https://download.lenovo.com/pccbbs/mobiles/r19fp02w.exe -O "$INSTALLER"
-shasum "$INSTALLER" | cut -d" " -f1 | cmp - "$HASH_FILE"
+#wget https://download.lenovo.com/pccbbs/mobiles/r19fp02w.exe -O "$INSTALLER"
+#shasum "$INSTALLER" | cut -d" " -f1 | cmp - "$HASH_FILE"
+
+mv $exe $INSTALLER
 
 #Extract the driver
 WINDRV="$TMP_DIR/windrv"
